presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -258,4 +262,15 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
 
         return Optional.of(new ConstraintApplicationResult<>(handle, constraint.getSummary()));
     }
+
+    private static ColumnMetadata makeColumnMetadata(String name, Type type)
+    {
+        return new ColumnMetadata(
+                name,
+                type,
+                null,
+                null,
+                false,
+                ImmutableMap.of(ORIGINAL_NAME, name));
```

Comment:
this should be

```
ColumnMetadata.builder()
  .setName(name)
  .setType(type)
  .setProperties(ImmutableMap.of(ORIGINAL_NAME, name))
  .builder();
```

Reply:
Oops. I missed this comment before I merged it. I'll update that and put up a separate PR.